android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -1232,9 +1232,23 @@
 		"strength": 50,
 		"cost": 300,
 		"requiredTech": "Combined Arms",
-		//"upgradesTo": "Helicopter Gunship",
+		"upgradesTo": "Helicopter Gunship",
 		"uniques": ["Bonus vs Armor 100%"]
 	},
+	{
+		// "Hovering unit" unique gives ability to get into impassable tiles - and only that (no vision bonus or flying over ocean)
+		// Unit embarks like any other ground unit and has penalty for attacking over river or from embarked state
+		// "Unable to capture cities"
+		"name": "Helicopter Gunship",
+		"unitType": "Melee",
+		"movement": 6,
+		"strength": 60,
+		"cost": 425,
+		"requiredTech": "Computers",
+		"requiredResource": "Aluminum",
+		"uniques": ["Bonus vs Armor 100%", "No defensive terrain bonuses", "Can move after attacking", "All tiles costs 1", "Can pass through impassable tiles", "Unable to capture cities"],
```

Comment:
Change to "All tiles cost 1 movement" if we're canonizing this

Reply:
should I still allow for "All tiles cost 1" for mods that are already using it?

Reply:
... Yeah, for now

Reply:
Done